### DISCUSSION TITLE
Skyline huawei p50 pro?

### DISCUSSION THREAD

**Discussion by Truenoxdx1** [skyline-emu/skyline#1490] [General]
can i use skyline on huawei p50 pro?

**Comment by Crytonics** [comment]
Do you have Snapdragon (Adreno) or Kirin (Mali) version? If you have Snapdragon (Adreno) then yes. If kirin (Mali) then you will be able play some games but Snapdragon is better for use Skyline.

**Reply by Truenoxdx1** [reply]
look at the huawei p50 pro it has the snapdragon 888 version but it doesn't have android it only has the harmonyos 2.0 version do you think it will hold without android?

**Reply by mateoalfaro** [reply]
> look at the huawei p50 pro it has the snapdragon 888 version but it doesn't have android it only has the harmonyos 2.0 version do you think it will hold without android?

Harmony os is still android,so atleast try to install the apk

**Reply by mateoalfaro** [reply]
Harmony os 2.0 is basically a10
